specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBSqlRoleDefinitionCreateUpdate.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "roleDefinitionId": "87717168-47d4-4dde-9021-25f683f6ec9d,",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "databaseAccountName": "contoso",
+    "api-version": "2020-06-01-preview",
+    "requestBody": {
+      "roleName": "Contoso Finance Records Reader",
+      "type": "CustomRole",
+      "assignableScopes": [
+        "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/dbs/contoso-sales",
+        "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/dbs/contoso-purchases"
+      ],
+      "permissions": [
+        {
+          "dataActions": [
+            "Microsoft.DocumentDB/databaseAccounts/databases/read",
+            "Microsoft.DocumentDB/databaseAccounts/databases/collections/read"
+          ]
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/cosmosRoleDefinitions/87717168-47d4-4dde-9021-25f683f6ec9d",
+        "name": "87717168-47d4-4dde-9021-25f683f6ec9d",
+        "type": "Microsoft.DocumentDB/databaseAccounts/cosmosRoleDefinitions",
+        "properties": {
+          "roleName": "Contoso Finance Records Reader",
+          "type": "CustomRole",
+          "assignableScopes": [
+            "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/dbs/contoso-sales",
+            "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/contoso/dbs/contoso-purchases"
+          ],
+          "permissions": [
+            {
+              "dataActions": [
+                "Microsoft.DocumentDB/databaseAccounts/databases/read",
+                "Microsoft.DocumentDB/databaseAccounts/databases/collections/read"
```

Comment:
Let's align with actions stated in this spec
https://microsoft.sharepoint.com/:w:/r/teams/DocumentDB/_layouts/15/Doc.aspx?sourcedoc=%7B55649B01-D9AD-4639-A256-E1B821AF5F5D%7D&cid=ca2d8bc7-c30c-4856-97d9-40fd5480c449